### ISSUE TITLE
Facing issue in prisma when using orderBy and cursor together in count method.

### ISSUE THREAD

**Issue by jaybhattjb** [prisma/prisma#13267]
Hi Team, 
My requirement is to get the count for the model based on where, orderBy, and cursor as below : 
findCountArgs : 
`
{
  "where": {
    "userId": 80,
    "isDeleted": false,
    "contentType": {
      "in": [
        "Image"
      ]
    }
  },
  "orderBy": [
    {
      "isPinned": "desc"
    },
    {
      "pinnedDate": "desc"
    },
    {
      "createdAt": "desc"
    }
  ],
  "cursor": {
    "id": 401
  }
}
`

`nextCount = await prismaModel.count({
      where: Object.assign({}, where),
      cursor: { id: cursor },
      orderBy
});`

But facing the below error when using cursor and orderBy together. 

Error : 
'Input error. Unable to process combination of query arguments for aggregation query. Please note that it is not possible at the moment to have a null-cursor, or a cursor and orderBy combination that not stable (unique).'

Can you please check and revert ASAP?

**Comment by richardwardza** [comment]
Does your `orderBy` contain any fields that are not marked as `unique` in your schema?

My reading of the error message is that if you use `cursor` and `orderBy` then your fields in the `orderBy` need to all be unique in your schema.

**Comment by jaybhattjb** [comment]
Yes, in "orderBy" using createdAt and a few other fields which are not unique.
